.spelling

```diff
@@ -213,6 +213,13 @@ _script.ps1
 _script.ps1.
  - demos/rest/README.md
 rest.ps1
+ - demos/Raspberry-Pi/README.md
+2.0.x
+2.1.x
+CoreClr
```

Comment:
`CoreCLR` is in the Global dictionary and I believe that is the correct "branding".

Reply:
Yup